src/lib/common/Kconfig

```diff
@@ -310,6 +310,10 @@ config SOL_BUS
 	default n
 
 menu "Hardware Options"
+config USE_AIO
+	bool "AIO Support"
```

Comment:
Yeah, I know. :)

It's used in other parts as well (flow node types, for instance), and it might be a good idea to use the same explanation there.